readme.md

```diff
@@ -105,6 +105,8 @@ Shortcut for `--arch=all --platform=all`
 
 `icon` - *String*
 
+For OS X, this must be `.icns` format.
+
```

Comment:
agree @ mention file format.

Oh yeah, I forgot #90 is how I got my linux apps to work. Does this work on all platforms as well? I know it wont update the binary logo on those systems, but it would be smart to always do that if so.